### DISCUSSION TITLE
The posibility to schedule an update at the first opportunity after all players have been disconnected.

### DISCUSSION THREAD

**Discussion by KissmanCZ** [gameserverapp/Platform#411] [Suggestions]
It would be great to be able to schedule updates so that they are done after the last player is disconnected, when no one is on the server. For example, at night some servers have a population drop to zero and the update could be done at that time.

**Comment by mvaessen** [comment]
Possible implications are to impactful to make this automated. I recommend using tasks to get this setup, when required task actions are available.
